Frameworks/UIKit.Xaml/ActivityIndicatorView.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ActivityIndicatorView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root">
+
+    <!-- TODO: Update property names in paths for binding once they have been updated in the header. See #2607 -->
+    <Canvas x:Name="indicatorCanvas"
+            Height="{Binding ElementName=Root, Path=ActivityIndicatorView_Height, Mode=OneWay}"
+            Width="{Binding ElementName=Root, Path=ActivityIndicatorView_Width, Mode=OneWay}"
+            HorizontalAlignment="Center"
+            VerticalAlignment="Center">
+        <ProgressRing x:Name="progressRing"
```

Comment:
Yeah, I also tried manually clearing the Clip property on Grid, but that didn't work either.

---
In reply to: [113579888](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113579888) [](ancestors = 113579888)